game/index.rb

```diff
@@ -0,0 +1,3 @@
+require_relative "./game"
+
+Game.new.start
```

Comment:
Normally ruby libraries are structured as: `lib/<name-of-the-library>`, in this case: `lib/tic-tac-toe`

Reply:
Thanks, I'll structure it that way